app/views/smart_proxies/logs/_modal.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="modal fade" id="backtrace-modal" tabindex="-1" role="dialog" aria-labelledby="bt-modal-label">
```

Comment:
Changed. How about this formatting?

![image](https://cloud.githubusercontent.com/assets/49752/12639319/fc23bd04-c5a3-11e5-9402-bdb784e7838b.png)

I don't want to make the dialog wider, it is set to this size for a reason. And log lines are all long. But in the new layout, there is much more room for those.